src/ControlSystem/ControlErrors/Size/ComovingCharSpeedDerivative.hpp

```diff
@@ -0,0 +1,347 @@
+ * \param horizon_00 the average coefficient of the horizon \f$\hat{S}_{00}\f$
+ * \param dt_horizon_00 the time derivative of horizon_00
+ * \param grid_frame_excision_sphere_radius radius of the excision boundary
+ *        in the grid frame, \f$r_{\mathrm{EB}}\f$.
+ * \param excision_rhat the direction cosine \f$\xi^\hat{i}\f$
+ * \param excision_normal_one_form the unnormalized one-form
+ *        \f$\hat{s}_\hat{i}\f$
+ * \param excision_normal_one_form_norm the norm of the one-form \f$a\f$
+ * \param frame_components_of_grid_shift the quantity
+ *        \f$\beta^i \frac{\partial x^\hat{i}}{\partial x_i}\f$
+ *        evaluated on the excision boundary.  This is not the shift in
+ *        the distorted frame.
+ * \param inverse_spatial_metric_on_excision_boundary metric in frame Frame.
+ * \param spatial_christoffel_second_kind the Christoffel symbols
+ *        \f$\Gamma^\hat{k}_{\hat{i}\hat{j}}\f$
+ * \param deriv_lapse the spatial derivative of the lapse
+ *        \f$\partial_\hat{i} \alpha\f$
+ * \param deriv_shift the spatial derivative of the shift
+ *        \f$\partial_\hat{j} \hat{\beta}^\hat{i}\f$. This is not the
+ *        derivative of frame_components_of_grid_shift.
+ *
+ * ## Background
+ *
+ * The characteristic speed on the excision boundary is
+ * \f{align}
+ *     v &= -\alpha + n_i\beta^i
+ * \f}
+ * where \f$\alpha\f$ is the lapse (invariant under frame transformations),
+ * \f$\beta^i\f$ is the grid-frame shift, and \f$n_i\f$ is the metric-normalized
+ * **outward-pointing** (i.e. pointing out of the black hole,
+ * toward larger radius)
+ * normal one-form to the excision boundary in the grid frame.
+ * (Note that the usual expression for the characteristic speed has
+ * a minus sign and defines \f$n_i\f$ as the inward-pointing (i.e. out of the
+ * computational domain) normal; here
+ * we have a plus sign and we define \f$n_i\f$ as outward-pointing because
+ * the outward-pointing normal is passed into comoving_char_speed_derivative.)
+ *
+ * For the size/shape map at the excision boundary,
+ * \f{align}
+ *   \hat{x}^i &= x^i (1 - \lambda_{00} Y_{00}/r_{\mathrm{EB}} + B),
+ *   \label{eq:map}
+ * \f}
+ * where \f$\hat{x}^i\f$ are the distorted-frame coordinates and \f$x^i\f$
+ * are the grid-frame coordinates.
+ * Here \f$Y_{00}\f$ is a
+ * spherical harmonic, \f$\lambda_{00}\f$ is
+ * the map parameter, and \f$r_{\mathrm{EB}}\f$ is the radius of the
+ * excision boundary in the
+ * grid frame (where the excision boundary is a sphere). Here
+ * \f$B\f$ represents terms that depend on angles but are independent of
+ * \f$\lambda_{00}\f$.  These \f$B\f$ terms depend on \f$\lambda_{\ell m}\f$ for
+ * \f$\ell>0\f$ but the precise form of these terms will not be important here,
+ * because below we will be differentiating the map with respect
+ * to \f$\lambda_{00}\f$.
+ *
+ * The comoving characteristic speed is
+ * \f{align}
+ *     v_c &= -\alpha -\hat{n}_\hat{i}\beta^\hat{i}
```

Comment:
Yes, it should be a plus sign here. Good catch!  The sign propagates to the last equation, eq. 28, where both shift terms should have signs switched [please make sure you agree]. There is another sign error in eq 28; the deriv-of-lapse term should have a plus sign because of the minus sign in eq. 12. [please make sure you agree]

Reply:
Yes in the final equation I agree that

1. The sign of the derivative of the lapse should should be switched from minus to plus
2. The signs of both the shift terms should be flipped from what they currently are.